internal/command/testdata/init-get/output.jsonlog

```diff
@@ -0,0 +1,8 @@
+{"@level":"info","@message":"Terraform 1.9.0-dev","@module":"terraform.ui","terraform":"1.9.0-dev","type":"version","ui":"1.2"}
+{"@level":"info","@message":"Initializing the backend...","@module":"terraform.ui","type":"init_output"}
+{"@level":"info","@message":"Initializing modules...","@module":"terraform.ui","type":"init_output"}
+{"@level":"info","@message":"- foo in foo","@module":"terraform.ui","type":"log"}
+{"@level":"info","@message":"Initializing provider plugins...","@module":"terraform.ui","type":"init_output"}
+{"@level":"info","@message":"","@module":"terraform.ui","type":"init_output"}
```

Comment:
This blank line message is an unfortunate artifact. Not a deal breaker but if it's an easy one to fix it'd be nice.